portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,7 +14,9 @@
 
 package com.google.sps.servlets;
 
+import java.util.*;
```

Comment:
instead of using a wildcard (`*`) import, let's import the specific class we need. This will clearly define the classes we need in addition to helping with readability. We can setup the IDE to avoid using * imports

Reply:
Sounds good! let me do that